build/InstrumentationLibraries.proj

```diff
@@ -1,6 +1,7 @@
 <Project>
   <ItemGroup>
     <SolutionProjects Include="..\**\OpenTelemetry.Instrumentation*.csproj" />
+    <TestProjects Include="..\test\**\OpenTelemetry.Instrumentation*.csproj" />
```

Comment:
Why add this? Did you mean to update the instrumentation libraries CI workflow to use this `InstrumentationLibraries.proj` file to run tests?

Reply:
I updated the workflow to call `dotnet test InstrumentationLibraries.proj --framework ${{matrix.version}}` but then it fails when some projects don't have a specific target (some libraries don't target `net462` for example) so I had to revert it. I left the test targets in `InstrumentationLibraries.proj` & `OpenTelemetry.proj` though because I thought it was nice to be able to do `dotnet test [project_name].proj` on the CLI for local things. But I can remove them if you want they aren't officially being called at the moment. My thinking is basically project files should define at least `Restore`, `Build`, and `VSTest` targets (that's what I'm doing [over here on contrib](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/build/Projects)).

Reply:
I think it's fine to have them defined.